examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -81,6 +81,8 @@ bool GenerateBdxUri(const Span<char> & fileDesignator, Span<char> outUri, size_t
 OTAProviderExample::OTAProviderExample()
 {
     memset(mOTAFilePath, 0, kFilepathBufLen);
+    queryImageBehavior = kRespondWithUpdateAvailable;
```

Comment:
Suggest default behavior be `NotAvailable` since it's not clear at construction time whether a file has been provided or not.

Reply:
Fixed